### PR TITLE
rampup: exercise to print "Hello OpenShift" to console

### PR DESCRIPTION
**Pull request by simrandhaliw**
Printed "Hello Openshift" to console
Containerized the application

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM golang:latest AS GOAPP
+WORKDIR /app 
+COPY file.go file.go
+RUN go build -o main .
+
+FROM ubi8  
+WORKDIR /app/
+COPY --from=GOAPP /app/main .
+ENTRYPOINT ["./main"]
+
```
Comment (ravisantoshgudimetla) [inline Dockerfile]: Empty lines?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM golang:latest AS GOAPP
```
Comment (ravisantoshgudimetla) [inline Dockerfile]: Why latest? Don't you want to pin it to a version?

File: exercises.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Exercise 0:
```
Comment (ravisantoshgudimetla) [inline exercises.txt]: Please combine the exercises and solutions files into a single readme

File: file.go [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package main
+import "fmt"
+func main() {
```
Comment (ravisantoshgudimetla) [inline file.go]: No space between imports and  functions

Comment (ravisantoshgudimetla) [reply file.go]: You should also run `gofmt` on this file.

File: mysolutionsteps.txt [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+(Note: your repo name path names will be different, so edit those accordingly to run above command)
+Next:  podman push quay.io/sdhaliwa/rampup
+
+Done!
+
+To check pulling and running your app from quay.io, follow these steps:
+podman pull quay.io/sdhaliwa/rampup
+
+podman run quay.io/sdhaliwa/rampup
+
```
Comment (ravisantoshgudimetla) [inline mysolutionsteps.txt]: Please remove extra lines

**Review by ravisantoshgudimetla — CHANGES_REQUESTED** [review]
Thanks for the PR @simrandhaliw. Please address the comments

**Review by ravisantoshgudimetla — APPROVED** [review]
LGTM

/approve

**Comment by simrandhaliw** [comment]
@ravisantoshgudimetla addressed to your comments. Please approve.

**Pull request closed without merging**